### PR TITLE
Bug: Fix exit command

### PR DESCRIPTION
**Pull request by bgopi23**
Resolves https://github.com/AY2324S2-CS2103T-T17-3/tp/issues/183

Screenshot 2024-03-27 at 8 43 53 PM

### CODE REVIEW

**Comment by dillontkh** [comment]
> @dillontkh do we need testcases for this?

Pinging Testing lead @RyanNgWH

File: src/main/java/seedu/address/logic/commands/ExitCommand.java [hunk 1]
```diff
@@ -9,7 +9,7 @@ public class ExitCommand extends Command {
 
     public static final String COMMAND_WORD = "exit";
 
-    public static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "Exiting Address Book as requested ...";
+    public static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "Exiting FitBook! This window will close in 5 seconds.";
```
Comment (dillontkh) [inline src/main/java/seedu/address/logic/commands/ExitCommand.java:12]:
```suggestion
    public static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "Exiting FitBook. Goodbye!";
```

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -175,9 +179,16 @@ void show() {
     private void handleExit() {
         GuiSettings guiSettings = new GuiSettings(primaryStage.getWidth(), primaryStage.getHeight(),
                 (int) primaryStage.getX(), (int) primaryStage.getY());
-        logic.setGuiSettings(guiSettings);
-        helpWindow.hide();
-        primaryStage.hide();
+
+        commandBox.freezeCommandBox();
+
+        Timeline timeline = new Timeline(new KeyFrame(Duration.seconds(5), exitEvent -> {
```
Comment (dillontkh) [inline src/main/java/seedu/address/ui/MainWindow.java:185]:
```suggestion
        Timeline timeline = new Timeline(new KeyFrame(Duration.seconds(2), exitEvent -> {
```

**Review by dillontkh — COMMENTED** [review]
Showing an exit message is a good idea! However, I feel like 5 seconds may be too long.

I tried reducing the timeout to 2 seconds, which felt just right in my opinion.

I have included the amendments as suggested changes.

**Comment by bgopi23** [comment]
2 seconds feels good to me too, commited your changes. thanks! @dillontkh

**Review by dillontkh — APPROVED** [review]
LGTM!

**Review by RyanNgWH — APPROVED** [review]
LGTM!

**Comment by RyanNgWH** [comment]
> @dillontkh do we need testcases for this?

Can't think of a good way to test this. Would be great if you have a way to test it, else it should be alright

**Pull request merged**